packages/x-scheduler/src/timeline/constants/units-config.ts

```diff
@@ -1,10 +1,10 @@
 import { TimelineView } from '@mui/x-scheduler-headless/models';
 
 export const TIME_UNITS_COUNT = 3;
-export const DAYS_UNIT_COUNT = 21;
-export const WEEKS_UNIT_COUNT = 8;
-export const MONTHS_UNIT_COUNT = 12;
-export const YEARS_UNIT_COUNT = 4;
+export const DAYS_UNIT_COUNT = 3 * 7;
```

Comment:
@noraleonte I made sure it takes at least 100% of the width on my 1440p screen, make the demos better looking 👌